charts/dependabot-gitlab/values.yaml

```diff
@@ -55,12 +55,12 @@ web:
   livenessProbe:
     # -- Enable liveness probe
     enabled: true
-    # -- Liveness probe failure thresold
+    # -- Liveness probe failure threshold
     failureThreshold: 5
     # -- Liveness probe period
-    periodSeconds: 10
+    periodSeconds: 60
     # -- Liveness probe timeout
-    timeoutSeconds: 1
+    timeoutSeconds: 10
```

Comment:
The timeout value for web service should remain low, either as is or maybe 2 seconds. With properly configured resources it should not take that long for web healthcheck to return status.